src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -123,14 +123,14 @@ uct_rc_mlx5_ep_short_dm(uct_rc_mlx5_ep_t *ep, uct_rc_mlx5_dm_copy_data_t *cache,
                         unsigned opcode, uint8_t fm_ce_se,
                         uint64_t rdma_raddr, uct_rkey_t rdma_rkey)
 {
-    uct_rc_mlx5_iface_t *iface = ucs_derived_of(ep->super.super.super.iface, uct_rc_mlx5_iface_t);
+    uct_rc_mlx5_iface_common_t *iface = ucs_derived_of(ep->super.super.super.iface, uct_rc_mlx5_iface_common_t);
     uct_rc_iface_t *rc_iface   = &iface->super;
     uct_rc_iface_send_desc_t *desc;
     void *buffer;
     ucs_status_t status;
     uct_ib_log_sge_t log_sge;
 
-    status = uct_rc_mlx5_common_dm_make_data(&iface->mlx5_common, &iface->super,
+    status = uct_rc_mlx5_common_dm_make_data(iface, &iface->super,
```

Comment:
now, no need to pass pure `RC` iface pointer, it can be obtained from `uct_rc_mlx5_iface_common_t`